provisioning/roles/cuttlefish-app/tasks/main.yml

```diff
@@ -92,3 +92,11 @@
     owner: root
     group: root
     mode: 0744
+
+- name: Create old email archiving cronjob
+  copy:
+    src: archiving
+    dest: /etc/cron.daily/archiving
```

Comment:
When will this run? Is it important that it runs at a particular time, e.g. when the server isn't heavily loaded?